checker/bin/spark-checker.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+LAUNCHER=
+# If debugging is enabled propagate that through to sub-shells
+if [[ "$-" == *x* ]]; then
+  LAUNCHER="bash -x"
+fi
+BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
+
+USAGE="Usage: alluxio-checker.sh Spark [SPARK_MASTER_ADDRESS] [PARTITIONS]
+The SPARK_MASTER_ADDRESS should be one of the following:
+  local or local[*] or local[K,F]  Running Spark on local machine.
```

Comment:
@aaudiber do we need to explain the local arguments more detail?
I think that most of our customers run Alluxio with spark cluster, so maybe we do not need to explain too many local arguments(the meaning of *, K, F).
Thanks!

Reply:
I agree - anyone who wants to use `local[*]` or `local[K,F]` should already understand them from the spark docs, so we don't need to explain here